docs/self-managed/setup/deploy/amazon/amazon-eks/eks-helm.md

```diff
@@ -181,139 +223,424 @@ spec:
 ```
 
   </TabItem>
+
 </Tabs>
 
+#### 2. Configure your deployment
+
+##### Enable Enterprise Components
+
+Some components are not enabled by default in this deployment. For more information on how to configure and enable these components, please refer to the official documentation:  
+[Configuring Enterprise Components and Connectors](../../../install.md#configuring-enterprise-components-and-connectors).
+
+##### Using internal Elasticsearch instead of the managed OpenSearch
+
+If you do not wish to use a managed OpenSearch service, you can opt to use the internal Elasticsearch deployment.
+This configuration disables OpenSearch and enables the internal kubernetes Elasticsearch deployment:
+
+<details>
+<summary>Show configuration changes to disable external OpenSearch usage</summary>
+
+```yaml
+global:
+  elasticsearch:
+    enabled: true
+  opensearch:
+    enabled: false
+
+elasticsearch:
+  enabled: true
+```
+
+</details>
+
+##### Using internal PostgreSQL instead of the managed Aurora
+
+If you prefer not to use an external PostgreSQL service, you can switch to the internal PostgreSQL deployment. In this case, you will need to configure the Helm chart as follows and remove certain configurations related to the external database and service account:
+
+<details>
+<summary>Show configuration changes to disable external database usage</summary>
+
+```yaml
+identityKeycloak:
+  postgresql:
+    enabled: true
+
+  # Remove external database configuration
+  # externalDatabase:
+  #   ...
+
+  # Remove service account and annotations
+  # serviceAccount:
+  #   ...
+
+  # Remove extra environment variables for external database driver
+  # extraEnvVars:
+  #   ...
+
+webModeler:
+  # Remove this part
+
+  # restapi:
+  #     externalDatabase:
+  #         url: jdbc:aws-wrapper:postgresql://${DB_HOST}:5432/${DB_WEBMODELER_NAME}
+  #         user: ${DB_WEBMODELER_USERNAME}
+  #         existingSecret: webmodeler-postgres-secret
+  #         existingSecretPasswordKey: password
+
+identity:
+  # Remove this part
+
+  # externalDatabase:
+  #     enabled: true
+  #     host: ${DB_HOST}
+  #     port: 5432
+  #     username: ${DB_IDENTITY_USERNAME}
+  #     database: ${DB_IDENTITY_NAME}
+  #     existingSecret: identity-postgres-secret
+  #     existingSecretPasswordKey: password
+```
+
+</details>
+
+##### Fill your deployment with actual values
+
+Once you've prepared the `values.yml` file, use the `envsubst` command to substitute the environment variables with their actual values. Run the following command:
+
+```bash
+# generate the final values
+envsubst < values.yml > generated-values.yml
+
+# print the result
+cat generated-values.yml
+```
+
+Starting from **Camunda 8.6**, you need to store various passwords in a Kubernetes secret, which will be consumed by the Helm chart. Below is an example of how to set up the required secret:
```

Comment:
`8.7` isn't it?
8.6 is just the deprecation message but 8.7 is enforcing.
We're also on the `next` docs, indicating 8.7